### ISSUE TITLE
Set "selector" for selections

### ISSUE THREAD

**Issue by stla** [rstudio/DT#983]
Hello,

  With the **Select** extension, there's an option [selector](https://datatables.net/reference/option/select.selector) to allow/disallow the selection on some columns. For example, if I have a `datatable` like this:

![dt_copycolumn](https://user-images.githubusercontent.com/4466543/159117883-cfd329eb-c436-4b75-837b-76a94ad0b538.png)

then when I click a 'Copy' button in the last column, this selects the clicked row, and this is not desirable. With the `selector` option I can prevent that.

Stop me if I'm wrong but it seems to me that there's no such possibility with the **DT** selection feature;

**Comment by shrektan** [comment]
You are right. At present, DT can only control whether certain cells are selectable or not. But what you need is to make certain cells not able to trigger the "select" action.
